source/user-guide/workflow/suggestion.rst

```diff
@@ -5,7 +5,7 @@ Suggestion for improving metadata content
 
 The metadata editor can be configured to analyse metadata and make suggestions to improve it. Examples of this include exploding a list of comma-separated keywords in a single element into multiple individual elements.
 
-The processes available in a given metadata schema are defined in the schema's ``suggest.xsl`` file, for example https://github.com/metadata101/iso19139.mcp/blob/master/src/main/plugin/iso19139.mcp/suggest.xsl
+The processes available in a given metadata schema are defined in the schemas ``suggest.xsl`` file, for example https://github.com/metadata101/iso19139.mcp/blob/master/src/main/plugin/iso19139.mcp/suggest.xsl
```

Comment:
The apostrophe is correct- it should be "schema's", or you could avoid the problem entirely by saying "schema" instead